src/Mod/Path/PathScripts/PathJob.py

```diff
@@ -142,10 +144,11 @@ def __init__(self, obj, models, templateFile=None):
         obj.PostProcessorArgs = PathPreferences.defaultPostProcessorArgs()
         obj.GeometryTolerance = PathPreferences.defaultGeometryTolerance()
 
-        ops = FreeCAD.ActiveDocument.addObject("Path::FeatureCompoundPython", "Operations")
+        #ops = FreeCAD.ActiveDocument.addObject("Path::FeatureCompoundPython", "Operations")
+        ops = FreeCAD.ActiveDocument.addObject("App::DocumentObjectGroup", "Operations")
         if ops.ViewObject:
-            ops.ViewObject.Proxy = 0
-            ops.ViewObject.Visibility = False
+        #     ops.ViewObject.Proxy = 0
+            ops.ViewObject.Visibility = True
```

Comment:
Is `True` not the default `Visibility` value?  If so, then this line may be removed all together, yes?